locales/th-TH/translation.json

```
@@ -0,0 +1,259 @@
+{
+    "$Language": {
+        "en": "English",
+        "pt-BR": "Português - Brasil",
+        "es": "Español",
+        "it": "Italiano",
+        "zh-TW": "繁體中文",
+        "de-DE": "Deutsch",
```

```suggestion
        "de-DE": "Deutsch",
        "hi": "हिंदी",
        "mr": "मराठी",
```

There are two languages missing here.

---

I just commit edited missing language. in [this commit](https://github.com/thamara/time-to-leave/pull/531/commits/a1c188740295e264ad2f087dde8ac879396f3c26)

Thanks for review.